nautobot/extras/templates/extras/dynamicgroup_edit.html

```diff
@@ -0,0 +1,26 @@
+{% extends "generic/object_edit.html" %}
+{% load form_helpers %}
+
+{% block form %}
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Dynamic Group</strong></div>
+        <div class="panel-body">
+            {% render_field form.name %}
+            {% render_field form.slug %}
+            {% render_field form.description %}
+            {% render_field form.content_type %}
+        </div>
+    </div>
+   {% if filter_form %}
+    <div class="panel panel-default">
+        <div class="panel-heading"><strong>Filter Fields</strong></div>
+        <div class="panel-body">
+            <span class="help-block">
+                Select the filtering criteria to determine membership of objects matching the Content Type for this Dynamic Group. Fields that are not a dropdown are expected to have string inputs and do not support multiple values.
+            </span>
+	    {% render_form filter_form %}
+        </div>
+    </div>
+    {% endif %}
```

Comment:
Might be helpful to have an `else` with some help text here, something like "Filtering criteria will be available after initially saving this group and returning to this page"?

Reply:
I would really like it to be dynamically rendered based on the `content_type`, similarly to how the parameters form for secrets are done. But this will do for now.

Reply:
<img width="755" alt="image" src="https://user-images.githubusercontent.com/138052/157969015-2e77d98b-f586-460d-b4c5-8604502509f9.png">